x-pack/plugins/ml/server/models/job_service/datafeeds.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+export function datafeedsProvider(callWithRequest) {
+
+  async function forceStartDatafeeds(datafeedIds, start, end) {
+    const jobIds = {};
+    const doStartsCalled = {};
+    const results = {};
+
+    datafeedIds.forEach((dId) => {
+      const jId = dId.replace('datafeed-', ''); // change this. this should be from a look up from the datafeeds endpoint
```

Comment:
Just looking at the comment, is this something you wanted to address in this PR?

Reply:
item added meta issue https://github.com/elastic/kibana/issues/20150